package.json

```diff
@@ -26,15 +28,22 @@
   "devDependencies": {
     "jasmine": "^3.1.0",
     "smee-client": "^1.0.1",
-    "standard": "^10.0.3"
+    "standard": "^10.0.3",
+    "nodemon": "^2.0.0"
   },
   "engines": {
-    "node": ">= 7.7.0",
-    "npm": ">= 4.0.0"
+    "node": ">= 8.3.0"
   },
   "standard": {
     "env": [
       "jasmine"
     ]
+  },
+  "nodemonConfig": {
+    "exec": "npm start",
+    "watch": [
+      ".env",
+      "."
+    ]
   }
-}
+}
```

Comment:
Add a newline (I would suggest you to config you editor to add a newline at the end of the file.)

Reply:
Done.
Thanks